docs/toml-schema.md

```diff
@@ -151,3 +151,36 @@ bors.some-repo.review = true
 # be set if `review` is also set.
 bors.some-repo.try = true
 ```
+
+## Repos
+
+Repos are configured by creating a file in the `repos` folder.
```

Comment:
Does the filename matter? I think either way we should suggest a pattern - maybe org-name if we don't want subdirectories.

Reply:
File names don't matter at this point. `org-name` might get confusing since the org and the name can include `-`. Perhaps `org#name` or is that too odd?

Reply:
@Mark-Simulacrum I implemented support for single-level directories.